src/util.f90

```diff
@@ -820,7 +820,7 @@ subroutine generate_bunch_bigaussian_weighted( &
  real(dp), allocatable :: ppcb_positions(:,:)
 
  bch=weight
- sigma_cut=3.0
+ sigma_cut=4.0
```

Comment:
Yes @cenit, this is what will certainly happen. We had to perform some tests before making sure for the best choice. Please let's keep 4 from now on. If different choices will be demonstrated to be equally important, we will put a control from nml.